.github/workflows/publish.yaml

```diff
@@ -28,3 +28,14 @@ jobs:
           docker login -u ${{ secrets.QUAY_USER }} -p ${{ secrets.QUAY_TOKEN }} quay.io
       - name: Build and push image
         run: make image push
+      - name: Tag manifests
+        if: startsWith(github.ref, 'refs/tags/')
+        run:
+          sed -i "s/main/${{github.ref_name}}/" $(git grep -l :main config/)
+        with:
```

Comment:
Is this needed?

Reply:
The manifests publish on the release has to use the image we are releasing, so "main" tag has to be replaced with "vx.y.z"

Reply:
Isn't it better to change the `VERSION` environment variable to the right tag, before creating the image?
`export VERSION=vx.y.z`
And do this only for tags

Reply:
> The manifests publish on the release has to use the image we are releasing, so "main" tag has to be replaced with "vx.y.z"

@qinqon Sorry, I meant to refer to the `with` attribute that seems empty

Reply:
Right, fixed.

Reply:
> Isn't it better to change the `VERSION` environment variable to the right tag, before creating the image? `export VERSION=vx.y.z` And do this only for tags

We are changing VERSION with 
```yaml
VERSION: ${{github.ref_name}}
```

The releas part of the publish is only done with a tag is pushed with 
```yaml
if: startsWith(github.ref, 'refs/tags/')
```

And since we are filtering tags with `- 'v*.*.*'` we are good there.